apps/sms/test/marionette/lib/views/conversation/accessors.js

```diff
@@ -0,0 +1,49 @@
+var SELECTORS = Object.freeze({
+  main: '#thread-messages',
+  message: '.message .bubble',
+  headerTitle: '#messages-header-text',
+  container: '#messages-container',
+  callButton: '#messages-call-number-button'
+});
+
+module.exports = {
+  get message() {
+    return client.helper.waitForElement(SELECTORS.message);
```

Comment:
I was asking the same question [in that comment](https://github.com/mozilla-b2g/gaia/pull/29631#discussion-diff-30245406) (which has been outraged). I don't see why the code is working.